src/silx/opencl/codec/byte_offset.py

```diff
@@ -41,7 +41,8 @@
 import functools
 import os
 import numpy
-from ..common import ocl, pyopencl
+from ..common import ocl  # noqa F401 Initialize OpenCL
```

Comment:
I keep all unused imports of `ocl`. I'm sure there is some monkey-business going on there.